submissions/jfrank/lib/index.js

```diff
@@ -0,0 +1,9 @@
+var server = require('./server')(8000)
+
+server.start(function(err) {
+  if (err) throw err
+
+  console.log('Server listening at "' + server.info.uri + '"')
```

Comment:
Minor style thing, I like doing this as `console.log('Server listening at "%s"', server.info.uri)`, but that's entirely personal preference.